cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/TransformContext.java

```diff
@@ -55,6 +55,12 @@
   Metrics getMetrics();
 
   /**
+   * Get the stage number of the transform, this is useful for setting the context for logging in transform.
+   * @return stage number
+   */
+  int getStageId();
```

Comment:
what about returning the PluginID.  Would have more info, including the stage id.